cmd/aws-iam-authenticator/root.go

```diff
@@ -119,6 +119,11 @@ func getConfig() (config.Config, error) {
 				logrus.Info("SSORoleMatch feature enabled")
 				config.SSORoleMatchEnabled = true
 			}
+			if strings.Contains(fg, string(config.ConfiguredInitDirectories)) &&
```

Comment:
Isn't there a utility function to do this?

Reply:
There does seem to be. I made a few changes there--the code looks nicer now.